R/romigami.R

```diff
@@ -14,4 +14,66 @@ omigami_init <- function(){
   omigami <- reticulate::import("omigami")
 
   return (omigami)
+}
+
+
+#' Generate a grid image representation of the hits returned from Spec2Vec and MS2DeepScore outputs.
+#' All structures passed MUST have valid smiles or inchi representations.
+#'
+#' @param spectra_matches DataFrame resulting from either Spec2Vec or MS2DeepScore. Need to feature smiles or inchi, score and compound_name as columns.
+#' @param representation The representation of the molecules found in the provided dataframe
+#' @param draw_indices If true draws the indices of the atoms
+#' @param molecule_image_size The size of every individual image of a molecule. Need to be provided as a list with two ints.
+#' @param substructure_highlight Needs to be a molecule substructure represented as a smiles.
+
+#' @return A image of the molecules
+#' @export
```

Comment:
if in the notebooks you are using
`plotter <- omigami$MoleculePlotter()`

you don't need any of this code on `romigami.R`

Reply:
this would be needed if you would be using the functions directly